modules/mri_violations/templates/menu_mri_violations.tpl

```diff
@@ -101,83 +101,9 @@
                         </tr>
                     </table>
                     <form method="post" action="{$baseurl}/mri_violations/"  name="mri_violations" id="mri_violations">
```

Comment:
This form was used to submit dropdown selections when they were changed, it does not work now because react table doesn't support this behaviour. As an alternative, you can make a POST on dropdown item change implemented in columnFormatter